controller_manager/src/controller_loader_pluginlib.cpp

```diff
@@ -16,15 +16,15 @@
 
 #include <memory>
 #include <string>
+#include <vector>
 
 namespace controller_manager
 {
 
 ControllerLoaderPluginlib::ControllerLoaderPluginlib()
-: ControllerLoaderInterface(),
-  loader_(std::make_shared<pluginlib::ClassLoader<controller_interface::ControllerInterface>>(
-      "controller_interface", "controller_interface::ControllerInterface"))
+: ControllerLoaderInterface("controller_interface::ControllerInterface")
```

Comment:
It was done like this initially by @Karsten1987 , I guess in case we could have other way of loading controllers without pluginlib.

We could see if it still makes sense.

Reply:
The original reasoning was to allow other types of loading mechanisms as well. I think we can keep it and perhaps mark it for a round of revision for the next ROS2 release?

Reply:
I have a better idea, wg meeting agenda point!

Reply:
https://github.com/ros-controls/ros2_control/issues/160

the council decided to remove it